app/assets/javascripts/kassi.js

```diff
@@ -337,9 +337,8 @@ function updateSellerGetsValue(priceInputSelector, displayTargetSelector, curren
     }
 
     $display.text(
-      [ST.paymentMath.displayMoney(Math.max(0, displaySum)),
-       $currency.val()]
-        .join(" "));
+      ST.paymentMath.displayMoney(Math.max(0, displaySum), currency, locale, false)
```

Comment:
By reading this like, the reader has no clue what the `false` might be. The reader needs to go to search the definition of the function to see that it means `inCents`.

To improve readability, you could either make a variable

```
var inCents = false;
ST.paymentMath.displayMoney( ..., inCents)
```

...or pass a options object:

```
var inCents = false;
ST.paymentMath.displayMoney( ..., {inCents: false})
```

I don't think this is a big issue so no need to change in this PR.